aries-site/src/pages/components/menu.mdx

```diff
@@ -45,9 +46,10 @@ An icon can be used on a menu item to provide additional context about the actio
 
 There are various places a Menu may be useful within an application. If the user is going to be selecting from a list of options to execute an action, Menu is a good component to use. The usage of labels and icons allow a menu to be versatile for different layout contexts.
 
-### Within a header
+### Within a header using a default Button
 
 A menu can be used in a [Header](/components/header) to group related actions.
+Use the default button variant when the menu has to be is placed in a row with other default buttons or with a heading.
```

Comment:
Wondering if we should be more prescriptive and say this instead:

The default Menu should be used in all instances unless the Menu is contained within a toolbar, in which case see [Menu within a toolbar](#menu-within-toolbar) guidance.

Reply:
good idea